decentraland/unity-renderer
# fix: map filter missing icons

This PR adds the icons for the layers friends and people on the filter menu. This way users have a visual hint to understand how this two layers look in the map.

![image](https://github.com/decentraland/unity-renderer/assets/51088292/73171676-4453-47ea-960a-fdaf649d8592)

How to test
1- Open this [link](https://play.decentraland.org/?explorer-branch=style/AddMapMissingIcns)
2- Open the nav map
3- Open the filters and check both icons are visible and the functionality is still working properly
